### DISCUSSION TITLE
Debugging Issue with CefSharp

### DISCUSSION THREAD

**Discussion by onlylau** [cefsharp/CefSharp#4945] [General]
### Debugging Issue with CefSharp Starting from Version 126.2.70

Hello everyone,

I am experiencing an issue with CefSharp in my Windows Forms project that began occurring after updating to version 126.2.70. Prior to version 127.3.50, CefSharp was functioning as expected, and I was able to debug my application normally. However, after updating to any version higher than 125.0.210, I encounter errors that prevent the creation of a debug file, thus hindering my ability to debug the application.

#### Problem Description:

1. **Observed Error**:
   When attempting to debug my application after updating CefSharp to versions later than 125.0.210, I receive error messages such as:
   ```
   [28844:6640:0826/142057.604:ERROR:device_event_log_impl.cc(196)] [14:20:57.603] USB: usb_service_win.cc:105 SetupDiGetDeviceProperty({{A45C254E-DF1C-4EFD-8020-67D146A850E0}, 6}) failed: Element not found. (0x490)
   ```
   Additionally, the debugging process does not function as expected, resulting in errors and the absence of the `debug.log` file.

2. **CefSharp Initialization Code**:
   The CefSharp initialization in my project is performed as follows:
   ```csharp
   CefSettings settings = new CefSettings
   {
       CachePath = Path.GetFullPath("cache"),
       // Other settings
   };

   Cef.Initialize(settings);
   CefWebBrowser = new ChromiumWebBrowser("https://example.com");
   CefWebBrowser.Parent = Dashboard;
   ```

3. **CefSharp Version**:
   - **Version That Works**: 125.0.210
   - **Problematic Versions**: Any version higher than 125.0.210

4. **Additional Observations**:
   - **Environment**: The project is a Windows Forms application using .NET Framework 4.8.
   - **Debugging Issue**: Unable to create the `debug.log` file to effectively investigate the problem.

#### Attempts at Resolution:

- Checked and updated CefSharp settings and code as needed.
- Attempted to create a simple new project to test if the problem persists.
- Consulted documentation and forums but did not find a specific solution for this issue.

#### Question:

Has anyone else encountered similar issues after updating to versions higher than 125.0.210 of CefSharp? If so, how did you resolve the problem? Is there any specific configuration or adjustment required for compatibility with newer versions of CefSharp?

Thank you in advance for your assistance and suggestions!

**Comment by amaitland** [comment]
How does the `MinimalExample` behave? I've not seen any reports that match yours.

https://github.com/cefsharp/CefSharp.MinimalExample

M128 is the current supported version. 

Make sure you have an `app.manifest` with `Win 10 compatibility` entry.

https://github.com/cefsharp/CefSharp/wiki/Quick-Start-For-MS-.Net-4.x#40-add-appmanifest-to-your-application
